ddlite.py

```diff
@@ -680,18 +679,18 @@ def __init__(self, candidates, feats=None, gt_dict=None):
     # LF data
     self.lf_matrix = None
     self.lf_names = []
-    # Model data    
-    self.X = None
-    self._w_fit = None
-    self.w = None
+    # Model probabilities
+    self.model = None
+    self.probabilities = None
+    self.lf_marginals = None
+    self.lf_weights = None
```

Comment:
`probabilities[i]` = P(y_i | \Lambda, x_i) where x_i are the features

The training name is ambiguous to me, I'm ok going with that for now

The weights are not accuracies. They're the preimage of the accuracies under the logit function